index.sh

```diff
@@ -54,29 +54,29 @@ platform=''
 machine=`uname -m`
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-  if [[ "$machine" == *"64" ]]; then
```

Comment:
I'm concerned about breaking existing setups that are returning something that matches `*"64"` but not `"x86_64"` -- Can we just move the `arm` branch to be the first `if` here, add the `aarch` check, and leave the `*"64"` checks alone?

Reply:
Sounds good to me, I updated the PR to check for ARM first.